webviz_subsurface/plugins/_tornado_plotter_fmu/shared_settings/_filters.py

```diff
@@ -0,0 +1,156 @@
+import json
+from typing import List, Optional
+
+import webviz_core_components as wcc
+from dash import ALL, Input, Output, callback, callback_context
+from dash.development.base_component import Component
+from webviz_config.webviz_plugin_subclasses import LayoutUniqueId, SettingsGroupABC
+
+from webviz_subsurface._providers import EnsembleTableProvider
+
+from .._plugin_ids import PlugInIDs
+
+
+class Filters(SettingsGroupABC):
+    """Settingsgroup for filters"""
+
+    class IDs:
```

Comment:
Place this together with the other ID's in `_plugin_ids.py`

Reply:
@HansKallekleiv  Here, I disagree. If we want the `SettingsGroup` to be reusable, the IDs should be in the same place as the settings group class. Hence, the best practice is to include them as a subclass. Do you see any issues with this way?

Reply:
No, I wasn't entirely sure of the practise here myself :) I just landed on the opposite side.
I have no issues with this.